scratch/stan_chol_exmpl.R

```diff
@@ -38,13 +39,20 @@ transformed parameters {
 }
 
 model {
+  //Priors
   to_vector(alpha) ~ std_normal();
   phi ~ uniform(0, 50);
   sigma_eps ~ gamma(sqrt(0.1),1);
+  sigma_obs ~ normal(0, 0.1); // Note entirely sure of the magnitude to use
+  // here,
 
   // Likelihood
+  // Assumes normally distributed observation error.
+  // obs_data are your vectors of draws the MVN at each time point (column
+  // bound to produce a matrix). The epsilon matrix are the
+  // mean of the expected realizations of the MVN draws.
   for (i in 1:n_times) {
-    obs_data[,i] ~ multi_normal(epsilon[,i], identity_matrix(n_sites));
+    obs_data[,i] ~ normal(epsilon[,i], sigma_obs);
```

Comment:
This all looks good, but I am a bit confused as to how `sigma_obs` matches with the centered stan model.  Why isn't this term on the centered stan model?

Reply:
The term isn't on the centered stan model because the values drawn from the MVN are being evaluated directly with the call `obs_data ~multinormal(0, Sigma)`

Whereas in the non-centered parameterization, we generate the expected values `epsilon` from the MVN using the cholesky decomposition transformation, and then we need to evaluate the observed data compared to the expected values `epsilon`, and this requires specifying a probability density (I chose normally distributed error but it could be anything, the point is just that we have to evaluate the log density of the observed data vs the expectation). 

I am going to post a simple example of a NCP and try to walk through it!  Also quite possible that I am over complicating this/made an error in this example, but I can't think of a way to do it without adding an additional `sigma_obs` term of some sorts. 

In our `wwinference` model, we already have an observation model and the expected values of the MVN get passed into another function and further transformed, so we don't have to add an extra parameter. It's just that in this simple example the MVN is our observation model.

Reply:
Yea, it could have just been a bad example on my part.  But I looked over the files you added and they look good, noncentered vs centered in stan makes a bit more sense to me now.

Reply:
No no was not a bad example on your part, I think it makes sense to default to the simplest model and I probably suggested it! I am adding one more example where the thing you non-center is a latent variable and am hoping it will make more sense!